mobile/lib/modules/home/ui/control_bottom_app_bar.dart

```diff
@@ -66,19 +76,35 @@ class ControlBottomAppBar extends ConsumerWidget {
                     )
                 : null,
           ),
-          ControlBoxButton(
-            iconData: Icons.archive,
-            label: "control_bottom_app_bar_archive".tr(),
-            onPressed: enabled ? onArchive : null,
-          ),
+          if (!hasRemote)
+            ControlBoxButton(
+              iconData: Icons.backup_outlined,
+              label: "Upload",
+              onPressed: enabled
+                  ? () => showDialog(
+                        context: context,
+                        builder: (BuildContext context) {
+                          return UploadDialog(
+                            onUpload: onUpload,
+                          );
+                        },
+                      )
+                  : null,
+            ),
+          if (hasRemote)
+            ControlBoxButton(
+              iconData: Icons.archive,
+              label: "control_bottom_app_bar_archive".tr(),
+              onPressed: enabled ? onArchive : null,
+            ),
         ],
       );
     }
 
     return DraggableScrollableSheet(
-      initialChildSize: 0.30,
+      initialChildSize: hasRemote ? 0.30 : 0.15,
       minChildSize: 0.15,
-      maxChildSize: 0.57,
+      maxChildSize: hasRemote ? 0.57 : 0.15,

```

Comment:
Why are these changed?

Reply:
Those are for the `DraggableScrollbarSheet` related change where if all the selections are only local assets, we shouldn't let the user drag the bottom sheet and only present them with the options available for local only assets. Setting the initial and maximum values to the same seemed like an easier way to achieve it